### PR TITLE
Add AssemblyMetadata[Language] attribute

### PR DESCRIPTION
**Pull request by vatsan-madhavan**
This PR addresses #587 

/cc @fadimounir

### CODE REVIEW

**Comment by vatsan-madhavan** [comment]
@fadimounir, Please note that this attribute will not be added to the satellite assemblies (resource assemblies like WindowsBase.resources.dll etc.).

File: eng/WpfArcadeSdk/tools/ExtendedAssemblyInfo.targets [hunk 1]
```diff
@@ -40,6 +40,26 @@
       <AssemblyMetadata Include="Commit" Condition="'$(SourceRevisionId)'!=''">
         <Value>$(SourceRevisionId)</Value>
       </AssemblyMetadata>
+
+      <!-- 
+        Crossgen wants to detect when an assembly is built from C++/CLI sources, 
+        since it is unable to function correctly today (ostensibly due to bugs). Since
+        /clr:pure is deprecated and WPF will eventually switch over to /clr (i.e., IJW),
+        Crossgen will eventually (i.e., in the near-term) fail to work on C++/CLI
```
Comment (fadimounir) [inline eng/WpfArcadeSdk/tools/ExtendedAssemblyInfo.targets]: Note: Crossgen does not support IJW today. There are no plans to support them so far

**Comment by fadimounir** [comment]
LGTM.

Thanks @vatsan-madhavan for adding this metadata. We've identified the root cause of the bug and are investigating a fix, but it's useful anyways to have this attribute.

**Comment by fadimounir** [comment]
> Please note that this attribute will not be added to the satellite assemblies (resource assemblies like WindowsBase.resources.dll etc.).

Yes. We don't crossgen those anyways

**Pull request merged**